search/include/pcl/search/pcl_search.h

```diff
@@ -38,7 +38,7 @@
  */
 
 #pragma once
-
+PCL_DEPRECATED_HEADER(1, 15, "Please include any needed headers from pcl/search directly.")
```

Comment:
No need to deprecate this. Just like the one in common

Reply:
Alright, removed it.